consensus/qbft/src/integration-test/java/org/hyperledger/besu/consensus/qbft/test/FutureRoundTest.java

```diff
@@ -68,11 +68,11 @@ public void messagesForFutureRoundAreNotActionedUntilRoundIsActive() {
     final ConsensusRoundIdentifier subsequentRoundId = new ConsensusRoundIdentifier(1, 6);
     final RoundSpecificPeers subsequentRoles = context.roundSpecificPeers(subsequentRoundId);
 
-    // required remotely received Prepares = quorum-2
+    // required remotely received Prepares = quorum-1
     // required remote received commits = quorum-1
 
     // Inject 1 too few Commit messages (but sufficient Prepare)
-    for (int i = 0; i < quorum - 3; i++) {
+    for (int i = 0; i < quorum - 2; i++) {
```

Comment:
the comment above this line is incorrect now, so should be removed.

Reply:
done